managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+Dialogs are the part of {{ site.data.product.title_short }} that you interface with when you order services.
+
+1. To open a service dialog, click **Automation** > **Embedded Automate** > **Customization**.
+2. From the *Service Dialog* section, review the list of created dialogs. Click one of the dialogs to view the details.
+3. Click **Edit the dialog**. You can edit the *Name*, *Provider*, and *Template*.
+
+   When you select to edit any of the dropdown settings, you can see a *dynamic* option. Toggle to **On** to make sure it is enabled.
+
+4. Click the **Options** tab. You can now select between *Embedded Automate* and *Embedded Workflows* in the Automation field.
+
+   ![Dialog Dynamic](../images/embedworkflow_servicedialogdynamic.png){: caption="Dialog Dynamic" caption-side="bottom"}
+
+   If you choose **Embedded Workflows**, you can select the workflow to run. Complete the fields, such as *Template* (using drag and drop functionality) and click **Save** when you complete the form. The workflow now runs and refreshes the fields with data from the workflow. In this way fields are chained together. When you *order* a service catalog, the workflow runs in the background to refresh the template.
+
+### Creating Service Catalog Items using an Embedded Workflow
+
+You can create a generic service catalog item that uses a created dynamic dialog and an embedded workflow. To use previously set up workflows and to view a service catalog item, you need to *order* the item. Complete the following steps to order a service catalog item that uses embedded workflows:
+
+1. Click **Services** > **Catalogs**.
+2. The *Service Catalogs* section opens and you see the existing Service Catalogs. If you want to order one, click it and then **Order**.
+
+   ![Service Dialog Dynamic](../images/embedworkflow_orderservicecatalog.png){: caption="Service Dialog Dynamic" caption-side="bottom"}
+
+   Ensure that the *VM Name*, *Provider* and *Template* fields are correct, and then click **Submit**.
+
+3. If you want to know more about the item first or to edit it, click the **Catalogs** section, and then click that item that is created. Click **Configuration** > **Edit this item**. For instance, one area that you can edit is the three service entry point fields. One field is for *Provisioning*, one for *Reconfiguring* and one for *Retirement*. You can now set these fields to *Embedded Workflows* instead of *Embedded Automate*. You can set the point field to the workflow that you want to use for any, or all, of the three service entry points.
+
+   ![Entry Points Dynamic](../images/embedworkflow_entrypointsembedded.png){: caption="Entry Points Dynamic" caption-side="bottom"}
+
+4. When you complete all your edits, click **Save**.
+5. Confirm that you can order the service catalog item and that it runs the workflows to drive the dynamic dialog dropdown.
+
+<!--
+##### Add docker images to a podified deployment and appliances (workflow registries)
+
+-->
```

Comment:
This probably belongs https://github.com/ManageIQ/manageiq-documentation/pull/1752/files#diff-a944ad335565b3b6f0c1776d24ce585d3b42531e2d45bc20a5eaab1950a871e4R281 we'll want to cover building docker containers (and probably point to other e.g. Docker documentation) as well as how to push to a registry and add the registry to the appliance/kubernetes system.

Reply:
I have moved this comment to the suggested location.